website/docs/development/contributing.md

```diff
@@ -39,6 +39,9 @@ The code need to pass verification by the following tools:
  - `flake8` : PEP8 compliance checker for Python, this includes copyright header verification
  - `isort .` : Ensure imports are sorted properly
  - `mypy --strict .` : Ensures code passes strict type checking
+ - `yamllint .` : Ensures that yaml files are syntactically correct and properly indented.
+
+`yamllint` will forbid the use of document start marker (`---`). When running locally, `yamllint` will run on all the yaml files, including the ones generated by hydra runs. We ignore the yaml files present in any `.hydra` directory.
```

Comment:
tmi.

```suggestion
 - `yamllint .` : Ensures that yaml files are well formatted and consistent
```

Reply:
Also, you said:
> When running locally, `yamllint` will run on all the yaml files, including the ones generated by hydra runs. We ignore the yaml files present in any `.hydra` directory.

This is not clear to me, besides .hydra, what other yaml files are generated?

Reply:
I think nothing else.

Reply:
I am removing the text you highlighted. Should I make a towncrier entry as well ?

Reply:
still seeing the text.